src/python/pants/backend/project_info/filter_targets.py

```diff
@@ -45,8 +48,18 @@ class FilterSubsystem(LineOriented, GoalSubsystem):
 
     target_type = StrListOption(
         metavar="[+-]type1,type2,...",
-        help="Filter on these target types, e.g. `resources` or `python_sources`.",
+        help=softwrap(
+            """
+            Filter targets based each targets's target type, e.g. `resources` or `python_sources`.
+
+            As with any target filter, a `-` prefix will negate matches for purposes of filtering;
+            that is, the filter will include a target only if the target's target type fails to
+            match all of the provided values.
+            """
+        ),
+        # help="",
```

Comment:
This needs to be removed at some point.

Reply:
https://github.com/pantsbuild/pants/pull/22391